submissions/bmukha/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,73 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/bmukha/a-tiny-JS-world
+   Web app: https://bmukha.github.io/a-tiny-JS-world/
+   */
+
+class Character {
+  constructor(specie, name, gender, saying, friends, legs) {
```

Comment:
You will benefit from setting default values (here and/or other classes' constructor definitions).
Also consider objects as arguments and object destructuring (which also allows to assign default values to missing props) in function props. This will give you greater flexibility in property management.